README.md

```diff
@@ -12,18 +12,19 @@ Linux, OSX and Windows are supported.
 This is a project initiated by Mozilla to gather code coverage results on Firefox.
 
 ## Table of Contents
+
 * [Usage](#usage)
-    * [LCOV output](#lcov-output)
-    * [Coveralls/Codecov output](#coverallscodecov-output)
-    * [grcov with Travis](#grcov-with-travis)
-    * [Auto Formatting](#auto-formatting)
+  * [LCOV output](#lcov-output)
+  * [Coveralls/Codecov output](#coverallscodecov-output)
+  * [grcov with Travis](#grcov-with-travis)
+  * [Auto Formatting](#auto-formatting)
```

Comment:
Could you revert the changes that are unrelated to this PR?

Reply:
done